index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="./css/MobileStyle.css">
+  <title>Portfolio</title>
+</head>
+<body>
+  <!--Hero Section Start-->
+
+  <section id="hero">
+    <header>
+      <div class="hamburger">
+        <a href="#"><i class="fa fa-bars"></i></a>
+      </div>
+
+      <nav id="navbar">
+        <ul class="manu">
+          <li><a href="index.html" id="portfolio">Portfolio</a></li>
+          <li><a href="#aboutme" id="about">About</a></li>
+          <li><a href="#form-section" id="contact">Contact</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <div class="banner">
+      <div class="introduction">
+        <section class="title">
+          <h1 class="self">
+            <span>Hey there.</span>
+            <span>I'm Shakil.</span>
+            <span>I’m a software developer</span>
+          </h1>
+        </section>
+
```

Comment:
- [ ] Good job inserting the container of your page , I will suggest you to use a `section` instead of the `div` for semantic reason 👍 
```bash
...
<section class="banner">
...
```